examples/non-monorepo/pages/_app.tsx

```diff
@@ -0,0 +1,6 @@
+import "../styles/globals.css";
+import type { AppProps } from "next/app";
+
+export default function App({ Component, pageProps }: AppProps) {
+  return <Component {...pageProps} />;
+}
```

Comment:
Do you want to just start with `app` dir + layouts enabled since it's new? (not sure if you needed something when generating the directory to make that happen)

Reply:
App directory is still beta and not recommended for production, so I think we should leave it out for now ([source](https://nextjs.org/blog/next-13))

Reply:
If we're going with that, `--turbo` should maybe _also_ be omitted.